### DISCUSSION TITLE
¿Que necesito saber para el parcial 2?

### DISCUSSION THREAD

**Discussion by JeanFranco99** [INGCOM-UNRN-P1/cursada-2023#193] [Preguntas y respuestas] [answered]
Buenas profes! Podrían hacer un debate como hicieron para el primer parcial? Con los temas necesarios para el 2do? Siento que el que se hizo para el primer parcial fue de mucha ayuda para terminar de repasar los temas. Saludos y gracias!

**Comment by Nikzza** [comment]
x2

**Reply by matiaspalleres** [reply]
x3

**Reply by martinvilu** [reply]
Usen esto
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/43c1f6bc-887c-4754-b048-cd2d5d8d237a)

En lugar de comentar, porque parece que se han contestado

**Comment by JeanFranco99** [comment]
@inomdedeu @martinvilu 
Por si no lo vieron, y si no es mucha molestia
Gracias 🫡

**Reply by martinvilu** [reply]
Molestia nunca es, pero también podían armar una ustedes y nosotros la comentamos.

**Comment by martinvilu** [comment, marked as answer]
Hay una parte reciclada de lo que escribió Iván para el primer parcial, pero los conceptos de memoria dinámica se superponen casi completamente.

### Arreglos y cadenas

 * ¿Entiendo como funcionan?
 * ¿Entiendo como se cargan, muestren y copian?
 * ¿Qué implica la capacidad de un arreglo?
 * ¿Cómo se pasa a una función?
 * ¿Entiendo la diferencia entre capacidad y el largo de una cadena?

### Matrices

 * ¿Entiendo como funcionan?
 * ¿Entiendo como se crean, cargan, muestren y copian?
 * ¿Cómo se pasa a una función?

### Punteros

 * ¿Entiendo como funcionan?
 * ¿Entiendo como se opera con ellos?
 * ¿Cómo se pasa a una función?
 * ¿En qué se diferencian de los arreglos?
 * ¿Cómo afectan el pasaje de argumentos?

### Archivos

 * ¿Entiendo como se abren?
 * ¿Entiendo como funcionan?
 * ¿Entiendo cuál es su ciclo de vida?
 * ¿Cómo sé cuando una operación falló? ¿Cómo puedo obtener detalles?

### Estructuras

 * ¿Entiendo como funcionan?
 * ¿Entiendo cuando se utilizan?
 * ¿Entiendo como se declaran y usan?
 * ¿Cómo se pasa a una función?

### Memoria dinámica

 * ¿Entiendo por qué existe aparte de la automática?
 * ¿Entiendo como se usa?
 * ¿Entiendo que es un ALV/VLA?
 * ¿Entiendo que es un puntero genérico?

### Argumentos por línea de comandos
 * ¿Entiendo como funcionan?
 * ¿Entiendo como se usan?

**Reply by JeanFranco99** [reply]
Gracias profe!
